### DISCUSSION TITLE
Difference between `type` and `let`?

### DISCUSSION THREAD

**Discussion by chandradeepdey** [FStarLang/FStar#3184] [Q&A] [answered]
Other than defining new inductive types, does `type` do anything `let` can't?

**Comment by chandradeepdey** [comment, marked as answer]
Ah, https://github.com/FStarLang/FStar/pull/3118

**Comment by nikswamy** [comment]
I tend to use `type` only when defining new inductive types and otherwise use `let`, or sometimes `let t : Type = d` instead of `type t = d`.

**Reply by chandradeepdey** [reply]
I usually do the same. Got confused when I saw something similar to #2933 working.
